prow/config/tide.go

```diff
@@ -28,6 +28,21 @@ type Tide struct {
 	// TODO: This will only be possible when we allow specifying orgs. At that
 	//       point, verify the above condition.
 	Queries []TideQuery `json:"queries,omitempty"`
+
+	// A key/value pair of an org/repo as the key and merge method to override
+	// the default method of merge. Valid options are squash, rebase, and merge.
+	MergeType map[string]string `json:"merge_method,omitempty"`
+}
+
+// MergeMethod returns the merge method to use for a repo. The default of merge is
+// returned when not overridden.
+func (t *Tide) MergeMethod(name string) string {
+	v, ok := t.MergeType[name]
+	if !ok {
+		return "merge"
```

Comment:
Can we enumerate the acceptable values for this in the `github` client package as constants?

Reply:
Will do. When I get feedback on the other parts I'll change this in the same change I update those other parts.